src/views/v2/tickets/item.php

```diff
@@ -117,6 +119,15 @@
 	$attributes['data-available-count'] = (string) $available_count;
 }
 
+$seat_labels = tribe( Session::class )->get_events_registrations_ticket_seat_label( $post_id, $ticket->ID );
```

Comment:
This cannot be done here. The Seating feature might or might be not registered and the ticket template should know nothing about it.

Filter the `$attributes` and hook on that from a Seating controller.

Reply:
Completely forgot about that -_- Changing it :D